neurom/features/morphology.py

```diff
@@ -428,36 +459,47 @@ def _mean_radius(neurite):
                     "values excluded all the points of the section so the radius of the first "
                     "point after the 'min_length_filter' path distance is returned."
                 )
-                # pylint: disable=invalid-unary-operand-type
+                # pylint: disable = invalid-unary-operand-type
```

Comment:
pylint handles spaces in the disable? I'd prefer to keep it the old way, if that's ok

Reply:
oops